document_merge_service/api/management/commands/clean_dangling_files.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from document_merge_service.api.models import Template
+
+
+class Command(BaseCommand):
+    help = "Remove dangling template files that are not attached to a template model anymore"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry-run", dest="dry", action="store_true", default=False)
+
+    def handle(self, *args, **options):
+        used_files = [template.template.path for template in Template.objects.all()]
+
+        for subdir, dirs, files in os.walk(settings.MEDIA_ROOT):
+            for f in files:
+                path = os.path.join(subdir, f)
+                if (
+                    path not in used_files
+                    and os.path.isfile(path)
+                    and path.endswith(".docx")
```

Comment:
Not sure this `endswith` restriction is actually needed

Reply:
Done